tachyon/zk/expressions/expression_unittest.cc

```diff
@@ -3,9 +3,10 @@
 #include "tachyon/base/random.h"
 #include "tachyon/math/finite_fields/test/finite_field_test.h"
 #include "tachyon/math/finite_fields/test/gf7.h"
+#include "tachyon/zk/expressions/evaluator/simple_selector_finder.h"
 #include "tachyon/zk/expressions/expression_factory.h"
 
-namespace tachyon::zk {
+namespace tachyon::zk::evaluator {
```

Comment:
I noticed that you created this `::evaluator` namespace, then removed or replaced it with the `::plonk` namespace everywhere except here. Could you remove all uses of `::evaluator`?

Reply:
Before moving the evaluators to plonk, it was good idea to put the evaluators into a separate namespace because tachyon::zk is a very big namespace.

Reply:
While that makes sense, it feels a bit unnatural that in the end, nothing else is in the namespace `::evaluator` except the code from [expression_unittest.cc](https://github.com/kroma-network/tachyon/pull/446/files/162adaf645efdd672ce92f2999ccf0852aa3ef09..60c52853429677b463e5fb6f3679d332f83bfe01#diff-8c0aa4d2ec5c60d99e1bb5ac0aab10faaaa21a434395d1468c17bacd14e950bf). Perhaps just removing it from there would suffice at least.

Reply:
Right I missed that :(

Reply:
I'll just remove all uses of `::evaluator` as you first suggested then.